src/components/ErrorFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { sites } from "../siteContent";
+
+interface Status {
+  status: string;
+  value: number;
+}
+
+const ErrorFilter = () => {
+  const [allStatus, setAllStatus] = useState();
+
+  useEffect(() => {
+    const getAllSitesStatus = async () => {
+      const urlList = Object.entries(sites).map(([, pageInfo]) =>
+        pageInfo.map((p) => p.url)
+      );
+      // Collect urls into one Array
+      const urls = ([] as any).concat.apply([], urlList);
+
+      const getSiteStatus = async (url: string) => {
+        const siteRes = await fetch("/proxy/" + url);
+        return siteRes.status;
+      };
+
+      const statusPriomises = [];
```

Comment:
the status promises can probably be done in a single line with a `.map` instead of a for of loop separately!

Reply:
also, `statusPromises` is misspelled!

Reply:
yep u r right! The use of `.map` is really smart!